content/06.methods.md

```diff
@@ -229,8 +229,9 @@ All p-values are two-sided unless otherwise stated.
 Z-scores were calculated using the formula $z=(x –\mu)/\sigma$ where $x$ is the value of interest, $\mu$ is the mean, and $\sigma$ is the standard deviation.
 
 #### Tumor purity (`tumor-purity-exploration` module)
-We used Theta2 (as described in the "Somatic Copy Number Variant Calling section" Methods section) to infer tumor purity of WGS samples and explored purity values across histologies. 
-We assumed that co-extracted RNA and DNA samples had the same tumor purity.
+Estimating tumor fraction from RNA directly is challenging since methods focus on estimating proportion of immune cells and assuming the rest is tumor [@doi:10.1038/ncomms3612], which is not a fair assumption for many of these diagnoses.
```

Comment:
```suggestion
Estimating tumor fraction from RNA directly is challenging because most assume tumor cells comprise all non-immune cells[@doi:10.1038/ncomms3612], which is not a valid assumption for many diagnoses in the PBTA cohort.
```